code/game/objects/effects/mines.dm

```diff
@@ -179,26 +173,26 @@
 /obj/effect/mine/stun/smart
 	name = "smart stun mine"
 	desc = "An advanced mine with IFF features, capable of ignoring people with mindshield implants."
-	smartmine = 1
-	disarm_time = 150
+	smartmine = TRUE
+	disarm_time = 15 SECONDS
 	disarm_product = /obj/item/deployablemine/smartstun
 
 /obj/effect/mine/stun/smart/adv
 	name = "rapid smart mine"
-	disarm_time = 80
+	disarm_time = 8 SECONDS
 	disarm_product = /obj/item/deployablemine/rapid
 
 /obj/effect/mine/stun/smart/heavy
 	name = "sledgehammer smart mine"
-	disarm_time = 170
-	stun_time = 230
+	disarm_time = 17 SECONDS
+	stun_time = 23 SECONDS
 	damage = 40
 	disarm_product = /obj/item/deployablemine/heavy
 
 
 /obj/effect/mine/stun/mineEffect(mob/living/victim)
 	if(isliving(victim))
-		victim.adjustStaminaLoss(stun_time)
+		victim.adjustStaminaLoss(damage)
```

Comment:
I asked you to rename the stun_time variable or leave the original stun behaviour....

This is not that...

Reply:
i reverted it